modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/provider/RedirectProviderImpl.java

```diff
@@ -156,6 +160,52 @@ protected void setRedirectPatternEntries(
 		_redirectPatternEntries = redirectPatternEntries;
 	}
 
+	private String[] _getCrawlerUserAgents() {
+		return new String[0];
+	}
```

Comment:
`crawlerUserAgents` is the name of the configuration that now has the `LayoutSEODynamicRenderingConfiguration` and will be merged with the new configuration that we will create. So in my opinion, this is better name because this will be the point where we get the values of the configuration. maybe this method will disappear.


I will put this on a different commit and comment that this is temporary and I will create an hardcore default value.

Reply:
Sounds good.